### PR TITLE
Use v1.0.1-beta.1 packages

### PR DESCRIPTION
**Pull request by johnbelamaric**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Uses the newer packages, which will use the newer images.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: e2e/provision/playbooks/roles/install/tasks/main.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@
...
-    - {pkg: porch-dev, version: porch-dev/v3, async: 480, poll: 5}
-    - {pkg: nephio-controllers, version: nephio-controllers/v5, async: 480, poll: 5}
-    - {pkg: configsync, version: configsync/v1, async: 240, poll: 5}
-    - {pkg: network-config, version: network-config/v1, async: 480, poll: 0}
-    - {pkg: nephio-stock-repos, version: nephio-stock-repos/v1, async: 60, poll: 0}
+    - {pkg: porch-dev, version: v1.0.0-beta.1, async: 480, poll: 5}
+    - {pkg: nephio-controllers, version: v1.0.1-beta.1, async: 480, poll: 5}
+    - {pkg: configsync, version: v1.0.1-beta.1, async: 240, poll: 5}
+    - {pkg: network-config, version: v1.0.1-beta.1, async: 480, poll: 0}
+    - {pkg: nephio-stock-repos, version: v1.0.1-beta.1, async: 60, poll: 0}
```
Comment (electrocucaracha) [inline e2e/provision/playbooks/roles/install/tasks/main.yml:23]: These packages require the creation of tags

**Comment by johnbelamaric** [comment]
Ok, we missed tagging nephio-resource-backend, which is an image we do not build in our current CI. I tagged it manually and will retest, and send a PR to update the instructions.

/test e2e

**Comment by johnbelamaric** [comment]
Missed one other image too (nephio-config-operator). Tagged now, retesting.

**Comment by electrocucaracha** [comment]
> Missed one other image too (nephio-config-operator). Tagged now, retesting.

But the failure was related to the tunnel NIC interface. Anyway, let's wait for the results

**Review by electrocucaracha — COMMENTED** [review]
/lgtm

**Pull request merged**